app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -517,6 +517,7 @@ class _CustomChatInputState extends ConsumerState<CustomChatInput> {
   }
 
   Future<void> onSendButtonPressed() async {
+    if(mentionKey.currentState!.controller!.text.isEmpty) return;
```

Comment:
arguably, if there is nothing to send, we do not only want to avoid that we want to disable the entire button, don't we?

Reply:
Actually, "onSendButtonPressed" is called from many places to send the message so that's why I have putted empty message condition here to avoid any black message from anywhere.